packages/rocketchat-livechat/server/models/Users.js

```diff
@@ -67,6 +67,45 @@ RocketChat.models.Users.findOnlineUserFromList = function(userList) {
  * Get next user agent in order
  * @return {object} User from db
  */
+RocketChat.models.Users.getBotAgent = function() {
+	const query = {
+		status: {
+			$exists: true,
+			$ne: 'offline'
+		},
+		statusLivechat: 'available',
+		roles: ['bot', 'livechat-agent']
```

Comment:
Yes! Thanks, I tried to search on the net the query model used here but failed, could you point me in the right direction?